dask/dataframe/backends.py

```diff
@@ -565,6 +581,37 @@ def group_split_pandas(df, c, k, ignore_index=False):
     return ShuffleGroupResult(zip(range(k), parts))
 
 
+def _union_categoricals_wrapper(
+    dfs: list[pd.CategoricalIndex] | list[pd.Series], **kwargs
+) -> pd.Categorical:
+    """
+    A wrapper around pandas' union_categoricals that handles some dtype issues.
+
+    union_categoricals requires that the dtype of each array's categories match.
+    So you can't union ``Categorical(['a', 'b'])`` and ``Categorical([1, 2])``
+    since the dtype (str vs. int) doesn't match.
+
+    *Somewhere* in Dask, we're possibly creating an empty ``Categorical``
```

Comment:
I'd like to avoid this if possible. I haven't found whose creating the empty categorical yet though :/